src/app.js

```diff
@@ -35,6 +35,34 @@ const mongo = mongoose.createConnection(String(process.env.MONGO_URL), {
 });
 
 async function main() {
+  const MAX = 28;
```

Comment:
yang harus di riset itu dia 30 msg/menit itu per `chatID` atau the whole app

Reply:
whole app, maxnya telegram 30 per menit, kalo per `chatID` bisa 30 nanti kalo 2 orang yg pake bisa dapet 429 lagi